src/test/lib/EventTest.cpp

```diff
@@ -299,10 +299,12 @@ QuicTestValidateConnectionEvents1(
 {
     TestScopeLogger ScopeLogger(__FUNCTION__);
 
-    MsQuicConfiguration ServerConfiguration(Registration, "MsQuicTest", ServerSelfSignedCredConfig);
+    MsQuicSettings Settings;
+    Settings.SetMinimumMtu(1280).SetMaximumMtu(1280);
```

Comment:
Why update all these tests to have a fixed MTU?

Reply:
Because otherwise we get datagram events every time we get a validated packet, which screw with the order of the events.

Reply:
Hmmm, ok.